app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java

```diff
@@ -41,7 +63,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         return binding.getRoot();
     }
 
+    @Override
+    public void onStop() {
+        super.onStop();
+        compositeDisposable.dispose();
+    }
+
     private void initView() {
         //
     }
+
+    private void renderContributors(List<Contributor> contributors) {
+        // TODO render contributors
```

Comment:
Sure.
As you said, Create `ContributorViewModel` which has some fields such as `name`, `imageUrl` and Callback. 
You can see [SessionViewModel](https://github.com/DroidKaigi/conference-app-2017/blob/master/app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java#L224).